src/System.Console/src/System/TermInfo.cs

```diff
@@ -458,6 +486,18 @@ private static short ReadInt16(byte[] buffer, int pos)
                      ((int)buffer[pos] & 0xff)));
             }
 
+            /// <summary>Read a 32-bit value from the buffer starting at the specified position.</summary>
+            /// <param name="buffer">The buffer from which to read.</param>
+            /// <param name="pos">The position at which to read.</param>
+            /// <returns>The 32-bit value read.</returns>
+            private static int ReadInt32(byte[] buffer, int pos)
```

Comment:
this could all be done in terms of spans and BinaryPrimitives.ReadInt32LittleEndian etc.
Probably not a good idea to do that right now, in case we need to port this change to servicing.

Reply:
That is true, we could do it like that but as you said, this might be ported to other branches where Span is not supported so It's probably better to just keep as is.

Reply:
Can you file an issue to update this to use the new APIs for master?